tests/unit_tests/test_wave_06.py

```diff
@@ -76,16 +76,26 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That the results is truthy
+    assert result
```

Comment:
I see that the prompt says to check that result is tuthy, but the description of the function is that it should return `True`, so I would rather literally compare against `True` here, as well as for the remaining tests (check for actual `True` or `False`).